aten/src/ATen/detail/MTIAHooksInterface.h

```diff
@@ -88,6 +90,15 @@ struct TORCH_API MTIAHooksInterface : AcceleratorHooksInterface {
   virtual void setCurrentStream(const c10::Stream& stream) const {
     FAIL_MTIAHOOKS_FUNC(__func__);
   }
+
+  virtual bool isPinnedPtr(const void* data) const override {
+    return false;
+  }
+
+  virtual Allocator* getPinnedMemoryAllocator() const override {
```

Comment:
FYI @egienvalue you can implement this if/when you need to support pinned host-side memory used for faster transfers to the device!

Reply:
That is nice. Right now we had a hacky way to pin CPU tensors.